.github/workflows/release.yml

```diff
@@ -17,6 +17,11 @@ on:
         description: "Default version to use when preparing a release."
         type: string
         required: true
+      releaseAssets:
```

Comment:
This is not yet used, maybe it should be removed for first implementing the dryRun functionality